judgels-backends/jophiel/jophiel-app/src/main/java/judgels/jophiel/user/superadmin/SuperadminCreatorConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package judgels.jophiel.user.superadmin;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.util.Optional;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableSuperadminCreatorConfiguration.class)
+public interface SuperadminCreatorConfiguration {
+    SuperadminCreatorConfiguration DEFAULT = new SuperadminCreatorConfiguration.Builder()
+            .enabled(true)
+            .username("superadmin")
+            .initialPassword("superadmin")
+            .initialEmail("superadmin@jophiel.judgels")
+            .build();
+
+    boolean getEnabled();
+    Optional<String> getUsername();
```

Comment:
Let's always use `superadmin` as username for simplicity. We never need to have another username.

Reply:
OK